### PR TITLE
Fix inconsistency in footer links

### LINKED ISSUE #5449: Inconsistency of links in footer
Links in the footer of https://readthedocs.org/ points to `latest` version of documentation except the link for `Documentation`, which points to https://docs.readthedocs.io/ which redirects to `stable` version.
~It should also points to https://docs.readthedocs.io/en/latest/~

All the links must points to `stable` version.

### PR DESCRIPTION
**Pull request by dojutsu-user**
Closes #5449

### CODE REVIEW

File: readthedocs/templates/base.html [hunk 1]
```diff
@@ -151,19 +151,19 @@ <h4>{% trans 'Get Involved' %}</h4>
 
           <ul>
             <li>
-              <a href="https://docs.readthedocs.io/en/latest/getting_started.html">{% trans 'Getting Started Guide' %}</a>
+              <a href="https://docs.readthedocs.io/page/getting_started.html">{% trans 'Getting Started Guide' %}</a>
```
Comment (stsewd) [inline readthedocs/templates/base.html]: Now this page redirects to `/intro/getting-started-with-sphinx.html`

Comment (dojutsu-user) [reply readthedocs/templates/base.html]: I have pushed the changes.
Thanks.

**Comment by dojutsu-user** [comment]
@davidfischer 
I would like to update this. There are many places where the link is pointed to `latest` version.
Will update this PR to point them at `stable` version.

**Comment by stsewd** [comment]
@dojutsu-user please open a new PR, so it's easy to review, I'm merging this one.

**Pull request merged**